modules/dop/dao/testplan_testcase_relation.go

```diff
@@ -161,7 +162,11 @@ func (client *DBClient) ListTestPlanCaseRels(req apistructs.TestPlanCaseRelListR
 		sql = sql.Where("`updater_id` IN (?)", req.UpdaterIDs)
 	}
 	if len(req.ExecutorIDs) > 0 {
-		sql = sql.Where("`executor_id` IN (?)", req.ExecutorIDs)
+		if strutil.Exist(req.ExecutorIDs, "unassigned") {
```

Comment:
explain what unassigned means?

Reply:
Searching test cases which have not been assigned to any executor yet, use "unassigned" to distinguish empty executors([]string) cases in params, any suggestions？